src/event/fetch.ts

```diff
@@ -0,0 +1,35 @@
+interface IFilter {
+  type: string;
+  contractIds: string[];
+  topics: string[][];
+}
+interface IEvent {
+  startLedger: string;
+  filters: Array<IFilter>;
+  pagination: object;
```

Comment:
Change `Array<IFilter>` to `IFilter[]`

make filters and paignation optional by adding ?